map/map_tf_generator/src/map_tf_generator_node.cpp

```diff
@@ -25,6 +27,8 @@
 #include <memory>
 #include <string>
 
+const int N_SAMPLES = 20;
```

Comment:
```suggestion
constexpr size_t NUM_SAMPLES = 20;
```

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/890/commits/c7d0b82975ec273e67d86b33b838afe70ceddf2d